x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/basic_license_essentials_tier/export_rules.ts

```diff
@@ -63,6 +66,24 @@ export default ({ getService }: FtrProviderContext): void => {
         expect(exportedRule).toMatchObject(ruleToExport);
       });
 
+      it('should export defaultable fields when values are set', async () => {
```

Comment:
This is in line with what @maximpn and @nikitaindik are doing ([example](https://github.com/elastic/kibana/pull/178295/files#diff-296d279d52abb0522bd61c2f8eaa44d2e51bedaed398b1645d681e14ce971d41R140-R166)) for the new fields. As discussed in the tech time meeting a few weeks ago, we wanted similar common logic to be added to a single test so that we aren't creating a new block of identical tests for each new field. [Here's a comment](https://github.com/elastic/kibana/pull/180638#discussion_r1565728806) with more info in the setup guide follow-up PR. So the adding of this is just so it lines up better when all the PRs are merged.

Reply:
Okay, sounds good 👍